analyzers/src/SonarAnalyzer.CSharp/Rules/ObjectShouldBeInitializedCorrectlyBase.cs

```diff
@@ -80,5 +83,20 @@ protected override void Initialize(SonarAnalysisContext context)
                         SyntaxKind.SimpleAssignmentExpression);
                 });
         }
+
+        protected static bool IsWebConfigAttributeSet(SonarAnalysisContext context, AnalyzerOptions options, string attribute)
```

Comment:
Because this class doesn't have `Cookie` in its name and the cookie path is hardcoded (that's a good thing) 🙁 
```suggestion
        protected static bool IsWebConfigCookieAttributeSet(SonarAnalysisContext context, AnalyzerOptions options, string attribute)
```